.happy/terraform/modules/service/main.tf

```diff
@@ -179,6 +182,10 @@ resource aws_ecs_task_definition task_definition {
         "name": "DATASETS_BUCKET",
         "value": "${var.datasets_bucket}"
       },
+      {
+        "name": "WMG_DISK_CACHE_PATH",
```

Comment:
This is a bit messy - you're adding a variable to a module that is not specifically tied to wmg (in fact, it is also used by the frontend). Unless you have specific reasons for keeping this parametric, I wouldn't mind hardcoding this in the dockerfile.

Reply:
@ebezzi - lol! That is what I had originally and changed to passing an environment variable. I had the same qualms as you but now that you mention it I will remove it entirely